codegen/method.go

```diff
@@ -77,10 +77,12 @@ type MethodSpec struct {
 	DownstreamService string
 	// The downstream method spec for the endpoint
 	DownstreamMethod *MethodSpec
-	// A map from upstream to downstream field names in the requests.
-	RequestFieldMap map[string]string
-	// A map from upstream to field names to downstream types in the requests.
-	RequestTypeMap map[string]string
+
+	// Statements for converting request types
+	ConvertRequestLines []string
```

Comment:
maybe change "Lines" to "GolangStatements" or something that makes it clear that these are lines of go code and the go code here is being build ahead of time by the convert function and not by the templates.